### ISSUE TITLE
Failure to decode empty json results to baking rights

### ISSUE THREAD

**Issue by ivanopagano** [Cryptonomic/Conseil#571]
Running latest master can fail to decode the json values from baking/endorsement rights, as shown in the following trace

```log
12:52:18.004 [lorre-system-akka.actor.default-dispatcher-5] ERROR  tech.cryptonomic.conseil.Lorre$      - Could not fetch blocks from client
com.fasterxml.jackson.databind.exc.MismatchedInputException: No content to map due to end-of-input
 at [Source: (String)""; line: 1, column: 0]
	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4145)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4000)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3042)
	at com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper.readValue(ScalaObjectMapper.scala:191)
	at com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper.readValue$(ScalaObjectMapper.scala:190)
	at tech.cryptonomic.conseil.util.JsonUtil$$anon$1.readValue(JsonUtil.scala:63)
	at tech.cryptonomic.conseil.util.JsonUtil$.fromJson(JsonUtil.scala:80)
	at tech.cryptonomic.conseil.tezos.TezosNodeOperator.$anonfun$getBakingRightsForBlock$1(TezosNodeOperator.scala:125)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:44)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
```

The obvious cause is using jackson object mapper without checking for possibly empty string inputs.

**Comment by ivanopagano** [comment]
This issue might be solved by work on #570 which could entail using circe decoders and additionally accept empty values as an expected result.

**Comment by ivanopagano** [comment]
Please check if this was actually solved by fixing #570 with PR #572 

@anonymoussprocket @vishakh @leenpaws
